templates/plugin/ansible.env.erb

```diff
@@ -0,0 +1,13 @@
+ANSIBLE_CALLBACK_WHITELIST="<%= @callback %>"
```

Comment:
should all these lines start with `export `? in my understanding bash won't pass the vars to child-processes otherwise.